consumers/sender/main.go

```diff
@@ -61,13 +61,20 @@ func (h *MessageHandler) LogFailedMessage(m *nsq.Message) {
 		return
 	}
 
+	logrus.WithFields(logrus.Fields{
+		"send_log_id":   msg.ID.String(),
```

Comment:
You don't need to invoke String() explicitly I think.

```suggestion
		"send_log_id":   msg.ID,
```